build.cake

```diff
@@ -233,6 +235,32 @@ Task("Create-Tools-Package")
             }
         });
     });
+
+Task("Create-Installer")
+    .IsDependentOn("Copy-Files")
+    .Does(() => {
+        CopyDirectory(binDir, installerDir + Directory("lib") + Directory("net45"));
+        CopyFileToDirectory("./src/Wyam.nuspec", installerDir);
+        var nuspec = GetFiles(installerDir + File("Wyam.nuspec")).FirstOrDefault();
+
+        NuGetPack(nuspec, new NuGetPackSettings
```

Comment:
Do we need to create a new package for wyam.exe here? We're already creating one using the nuspec at src\Wyam\Wyam.nuspec up in the Create-Tools-Package task. Could the Create-Installer task depend on Create-Tools-Package and use that resulting package? In other words, curious what the difference is between the "tools" package and the one we create for Squirrel?